eagle-stores/src/utils/actionType.js

```diff
@@ -0,0 +1,6 @@
+export const ACTION_TYPE = {
+    INITIALIZE_PRODUCTS : "INITIALIZE_PRODUCTS",
+    INITIALIZE_CATEGORIES: "INITIALIZE_CATEGORIES",
+    filterChange: "filterChange",
```

Comment:
better to follow only one convention on constants. Don't use different Convensions

Reply:
Done